### PR TITLE
Contributing.md

### LINKED ISSUE #3: Add Contributing Guidelines

### PR DESCRIPTION
**Pull request by kdsinghcoder**
# Issue Related with PR
- fixes #3 

# Proposed Changes
- Change 1
- Change 2

# Additional info if necessary
- any additional information

# Checklist
- [x] Tests
- [x] Resolve Conflicts

# Screenshots

Original    |   Updated
:----------------: | :----------------:
** Original SS ** | ** Updated SS **

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Report bugs using Github's [issues](https://github.com/briandk/transcriptase-atom/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
+
+## Write bug reports with detail, background, and sample code
+[This is an example](http://stackoverflow.com/q/12488905/180626) of a bug report I wrote, and I think it's not a bad model. Here's [another example from Craig Hockenberry](http://www.openradar.me/11905408), an app developer whom I greatly respect.
```
Comment (YashKumarVerma) [inline .github/CONTRIBUTING.md:25]: avoid using personal pronouns like `I` in this

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [My stackoverflow question](http://stackoverflow.com/q/12488905/180626) includes sample code that *anyone* with a base R setup can run to reproduce what I was seeing
```
Comment (YashKumarVerma) [inline .github/CONTRIBUTING.md:32]: same, avoid MY

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
@YashKumarVerma fixed as per requirement
.

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**